modules/server/src/mapping/extendCharts.ts

```diff
@@ -0,0 +1,18 @@
+export const extendCharts = (config, extendedFields) => {
+	/**
+	 * TODO: extend with gql types
+	 */
+
+	/**
+	 * TODO: extend with gql query
+	 */
+
+	/**
+	 * TODO: validate displayNames match the fields given
+	 */
+
+	/**
+	 * TODO: validate the chart type supports the GQL field type
+	 */
+	return config;
+};
```

Comment:
placeholder to follow current config pattern, with future functionality marked appropriately with TODO